dt-backend/src/model/war-log-entry/war-log-entry.service.ts

```diff
@@ -5,11 +5,6 @@ import { WarLogEntry } from "../types/war-log-entry.type";
 @Injectable()
 export class WarLogEntryService {
   async getWarLogEntries(): Promise<WarLogEntry[]> {
-    return prisma.warLogEntry.findMany({
-      include: {
-        attacker: true,
-        defender: true,
-      },
-    });
+    return prisma.warLogEntry.findMany();
```

Comment:
We'll need to think about this one! In this case, we definitely need at least of both properties... Best would be - both!  
Since a Warlog Entry has some data that's gonne be displayed. Assuming you're the attacker, then you gotta know the defender as soon as you open the Warlog Entry!  
So yeah, we'll have to think about this one!